docs/self-hosted/maintenance.md

```diff
@@ -1,6 +1,18 @@
 # Maintenance
 ## Private Packagist Self-Hosted (Replicated Native)
 
+This guide explains how to setup Private Packagist Self-Hosted (Replicated Native) which is deprecated.  
```

Comment:
Would maybe make sense, to make this more in-your-face:
```suggestion
DEPRECATED: This guide explains how to setup the now deprecated Private Packagist Self-Hosted (Replicated Native).  
```
What do you think?

Reply:
I think its in your face enough, because there is the hint below with redmark next to it. But I like the other part of the suggestion, to move this before the long name